plugin/callgraph/builtins.go

```diff
@@ -0,0 +1,42 @@
+package callgraph
+
+import (
+	"embed"
+	"encoding/json"
+
+	"github.com/safedep/code/core"
+	"github.com/safedep/dry/log"
+)
+
+//go:embed builtins.json
+var builtinsFS embed.FS
+
+// languageBuiltins holds built-in functions for each language
+type languageBuiltins map[string][]string
+
+var allBuiltins languageBuiltins
+
+// LoadBuiltins loads built-in functions from the embedded JSON file
+func init() {
```

Comment:
Can we avoid doing an `init` here. CAF is a framework really. It may be used for other plugins and not just CallGraph. Loading all the builtins in memory for any app that uses CAF seems unnecessary.

You can have a the same logic in `initBuiltins` which need to be explicitly called.

You can use `sync.Once(..)` to execute the builtin loader once when a Callgraph plugin is created (using NewXXX)`